src/app/components/PrivateRoute/index.js

```diff
@@ -9,7 +9,11 @@ function PrivateRoute({ component: Component, path }) {
     <Route
       path={path}
       render={props =>
-        sessionStorage.getItem('user') ? <Component {...props} /> : <Redirect to={ROUTES.LOG_IN()} />
+        sessionStorage.getItem('authorizationToken') ? (
```

Comment:
move this function to `utils/sessionStorage`. `getAuthToken()` `setAuthToken(token)`